### PR TITLE
fix: Normalize selector

### PR DESCRIPTION
**Pull request by bbernays**
🎉 Thank you for making CloudQuery awesome by submitting a PR 🎉

#### Summary

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/.github/CONTRIBUTING.md) 🧑‍🎓
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run --new-from-rev main` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: pkg/policy/execute.go [hunk 1]
```diff
@@ -245,15 +245,24 @@ func (e *Executor) Execute(ctx context.Context, req *ExecuteRequest, policy *Pol
 	return &total, nil
 }
 
-func (e *Executor) createCheckResult(ctx context.Context, selector []string, policyExecution *state.PolicyExecution, q *Check, qr *QueryResult, err error) error {
+func normalizeCheckSelector(policyExecution *state.PolicyExecution, policyPath []string, policyName string) string {
```
Comment (disq) [inline pkg/policy/execute.go:248]: Bare functions (`normalizeCheckSelector`) should be towards the end of the file

Comment (bbernays) [reply pkg/policy/execute.go]: fixed

File: pkg/policy/execute.go [hunk 2]
```diff
@@ -245,15 +245,24 @@ func (e *Executor) Execute(ctx context.Context, req *ExecuteRequest, policy *Pol
...
+	selector := []string{policyExecution.PolicyName}
+	if !strings.HasPrefix(selector[0], policyExecution.Location+"//") {
+		selector[0] = policyExecution.Location + "/"
```
Comment (disq) [inline pkg/policy/execute.go]: Not sure I get the logic of overwriting policyName here, and then appending it in L254 regardless of if it was overridden or not.

Comment (bbernays) [reply pkg/policy/execute.go]: Good catch. I made a mistake in naming... `policyName` should be `checkName`

**Pull request merged**